cves/CVE-2011-3192.yml

```diff
@@ -103,10 +106,15 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
+  answer: |
+    The vulnerability was originally identified by Google Security Research, Michal
+    Zalewski in January 2007. It was eventually exploited in the wild and rose 
+    to fame in 2011 as the Apache Killer. Four years passed between the discovery of 
+    this vulnerability and the resolving patch, but in there were no major events
```

Comment:
**Grammar**
... this vulnerability and the resolving patch, but `in between` there were no major events

Reply:
Updated.